### DISCUSSION TITLE
Enable SyslistView32 (syslistview32-enabler) Mod: Causes Return of Full-Line Select, Leaving Nowhere to Click that Doesn't Grab an Existing Item.

### DISCUSSION THREAD

**Discussion by FLskydiver** [ramensoftware/windhawk-mods#666] [General]
When browsing a folder with a great many subfolders or items, when Enable SysListView32 is enabled, clicking empty space in a folder utilizing the LIST view using one's mouse becomes effectively impossible, without scrolling all the way to the end of (why might be a very long) list where (hopefully) there should be some free white-space.  

By Default, Windows 10 still uses Full-Row Select (which remains mildly annoying) but at least in LIST view, there is a small amount of padding between rows where one can click without activating a new item.

This is very important because one often needs to click white-space to deselect a previous selection; access the folder context menu; or specify the folder as the target destination of a copy or move operation and not one of the sub-folders which otherwise may almost invariably be selected.

Can anything be done about this issue; or is it beyond the scope of the tweak?

Thank You!

**Comment by m417z** [comment]
Tagging @anixx, the mod author.

**Comment by Anixx** [comment]
Does this happen if you enable SysListView32 via registry? https://files.catbox.moe/kfr9x4.reg

**Reply by FLskydiver** [reply]
Thank you for your prompt assistance!

Yes, it still does.  But the keys you provided didn't change anything in my system from what already existed:  Here is how those two keys looked before, and how they look now:

```
Windows Registry Editor Version 5.00

[HKEY_CLASSES_ROOT\CLSID\{1eeb5b5a-06fb-4732-96b3-975c0194eb39}]
@="Itemsview"

[HKEY_CLASSES_ROOT\CLSID\{1eeb5b5a-06fb-4732-96b3-975c0194eb39}\InProcServer32]
@=hex(2):25,00,53,00,79,00,73,00,74,00,65,00,6d,00,52,00,6f,00,6f,00,74,00,25,\
  00,5c,00,73,00,79,00,73,00,74,00,65,00,6d,00,33,00,32,00,5c,00,65,00,78,00,\
  70,00,6c,00,6f,00,72,00,65,00,72,00,66,00,72,00,61,00,6d,00,65,00,2e,00,64,\
  00,6c,00,6c,00,00,00
"ThreadingModel"="Apartment"

[HKEY_CLASSES_ROOT\WOW6432Node\CLSID\{1eeb5b5a-06fb-4732-96b3-975c0194eb39}]
@="Itemsview"

[HKEY_CLASSES_ROOT\WOW6432Node\CLSID\{1eeb5b5a-06fb-4732-96b3-975c0194eb39}\InProcServer32]
@=hex(2):25,00,53,00,79,00,73,00,74,00,65,00,6d,00,52,00,6f,00,6f,00,74,00,25,\
  00,5c,00,73,00,79,00,73,00,74,00,65,00,6d,00,33,00,32,00,5c,00,65,00,78,00,\
  70,00,6c,00,6f,00,72,00,65,00,72,00,66,00,72,00,61,00,6d,00,65,00,2e,00,64,\
  00,6c,00,6c,00,00,00
"ThreadingModel"="Apartment"
```

Might there be a possibility of toggling this mod on and off globally with a hotkey, (or on a per-window basis using something like autohotkey as an intercept), if we can't get the exclusion issue sorted?
